simple_utils.py

```
@@ -21,3 +21,6 @@ def calcular_promedio(lista_numeros):
         suma += numero
     promedio = suma / len(lista)  # Error: 'lista' no está definido
```

Ruff (0.11.9)

Undefined name `lista`

(F821)